charts/rasa-x/templates/rasa-open-source-ingress.yaml

```
@@ -42,9 +73,14 @@ spec:
           serviceName: {{ $fullName }}-{{ $.Values.rasa.versions.rasaProduction.serviceName }}
           servicePort: {{ $.Values.rasa.port }}
         path: /webhooks/
+      - backend:
+          serviceName: {{ $fullName }}-app
+          servicePort: {{ default 5055 $.Values.app.port }}
+        path: /app/(.*)
```

```suggestion
      - backend:
          serviceName: {{ $fullName }}-{{ $.Values.rasa.versions.rasaProduction.serviceName }}
          servicePort: {{ $.Values.rasa.port }}
        path: /core/(.*)
```

Why here is the app edpoint?

---

Ahh my bad this is redundant code. Not required 🙈 `/core` endpoint is already defined `L:37` removing this unnecessary.

---

done removed